web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -682,7 +682,7 @@
             />
           {/if}
           {#if $slideshowState === SlideshowState.None && isShared && ((album && album.isActivityEnabled) || numberOfComments > 0)}
-            <div class="z-[9999] absolute bottom-0 right-0 mb-4 mr-6">
+            <div class="z-[9999] absolute bottom-0 right-0 mb-12 mr-4 justify-self-end">
```

Comment:
I don't think `justify-self-end` works here. What browser is your screenshot from? Getting the bottom spacing right is pretty difficult, because there's a lot of variability between browsers and even within the same browser when zooming.

Reply:
The browser is windows chrome, same as today's screenshots (the video controls yesterday changed owing to videos consistently failing to load). `justify-self-end` is imported from reverting the change that introduced the black bezels (#8757), but I can look further into it.

Reply:
I am not noticing a difference from removing `justify-self-end` using dev tools so it was possibly a trivial fix in the aforementioned change.

Reply:
Zooming seems to keep things consistent on windows chrome, though things overlap more in all cases since the usual video control style came back to my setup, so maybe I would advise switching to row 2.5 rather than 2 from the bottom (where row 1 is by definition symmetric to the top row).

![image](https://github.com/immich-app/immich/assets/28405293/a4801e1a-d6f3-4ab7-8ab0-cb20cc083063)

![image](https://github.com/immich-app/immich/assets/28405293/c20d2bac-974b-428d-8112-5a70b12d511b)

Reply:
Windows firefox looks superb on pr branch at all zoom levels (within reason haha).
![image](https://github.com/immich-app/immich/assets/28405293/f3d1e963-9422-491f-924b-abd9a57dea05)
![image](https://github.com/immich-app/immich/assets/28405293/fcdaa17e-b981-43ae-8993-e725286c3542)
My iPhone won't connect to the Vite server unfort.